content/en/xtc/loadtesting/170-monitor-lt.md

```diff
@@ -24,8 +24,23 @@ Steps that failed due to errors (like timeouts or failed authentication) will be
 
 ### Scenario Status
 
-The _Scenario Status_ tab only contains information while the test is actually running or finished. This is an overview of all test scenarios that are executed in this load test, containing scenario name and state and other useful information such as currently running users, average scenario runtime and especially the number of events and errors that occurred in this scenario:
+The _Scenario Status_ tab only contains information while the test is actually running or finished. After the [live metrics]({{< relref "#live-metrics" >}}) follows an overview of all test scenarios that are executed in this load test, containing scenario name and state and other useful information such as currently running users, average scenario runtime and especially the number of events and errors that occurred in this scenario:
 
 {{< image src="xtc/loadtest_scenarioStatus.png" >}}
 The scenario status view for a running load test.
-{{< /image >}}
\ No newline at end of file
+{{< /image >}}
+
+#### Live Metrics
+
+The graph on top of the scenario status table is updated live while the load test is running and displays information on the number of currently active users for each test scenario as well as total users. Hovering over the graph displays a popup of the values for this point in time.
+
+Since this might be a lot of information, you can filter in the dropdown for single or several test scenarios or just display total values. 
+
+Red vertical lines are displayed when scenario errors occurred. These are bound to the respective scenarios, i.e. if you select a single scenario only the errors for this scenario will be visible in the graph. If you select _Totals_, all errors will be displayed.
```

Comment:
```suggestion
Red vertical bars are displayed when scenario errors occurred. These are bound to the respective scenarios, i.e. if you select a single scenario only the errors for this scenario will be visible in the graph. If you select _Total_, the sum of all scenario errors will be displayed.
```

Reply:
The item in the dropdown says 'Totals' - unless you changed that recently...

Reply:
We changed that recently.   :-)